### PR TITLE
Add lead_team, teams and disabled_on filter to event search.

### PR DESCRIPTION
**Pull request by elcct**
As in the title.

### CODE REVIEW

File: datahub/search/event/views.py [hunk 1]
```diff
@@ -12,22 +12,28 @@ class SearchEventParams:
     serializer_class = SearchEventSerializer
 
     FILTER_FIELDS = (
+        'address_country',
+        'disabled_on_after',
```
Comment (reupen) [inline datahub/search/event/views.py]: Do these cover 'disabled_on is not null'?

File: datahub/search/event/tests/test_views.py [hunk 2]
```diff
@@ -149,6 +145,83 @@ def test_search_event_address_country(self, setup_es, setup_data):
...
+
+    def test_search_event_teams(self, setup_es, setup_data):
+        """Tests teams filter."""
+        url = reverse('api-v3:search:event')
+
+        team_a = TeamFactory()
+        team_b = TeamFactory()
+        for _ in range(5):
+            EventFactory(
+                teams=[team_a]
```
Comment (reupen) [inline datahub/search/event/tests/test_views.py]: Would be nice to have some events with multiple teams here as well.

File: datahub/search/event/views.py (hunk 1 above)
Comment (elcct) [reply datahub/search/event/views.py]: This is a very good point. I have added `disabled_on_exists` filter to cover this case. This one accepts boolean as a parameter and is useful to get just disabled events or only active.

File: datahub/search/event/tests/test_views.py [hunk 3]
```diff
@@ -149,7 +145,133 @@ def test_search_event_address_country(self, setup_es, setup_data):
...
+    def test_search_event_lead_team(self, setup_es):
+        """Tests lead_team filter."""
+        url = reverse('api-v3:search:event')
+
+        team = TeamFactory()
+        for _ in range(5):
+            EventFactory()
+            EventFactory(
+                lead_team_id=team.id
+            )
```
Comment (marcofucci) [inline datahub/search/event/tests/test_views.py]: isn't this the same as this?
```
EventFactory.create_batch(5)
EventFactory.create_batch(5, lead_team_id=team.id)
```

File: datahub/search/event/tests/test_views.py (hunk 2 above)
Comment (elcct) [reply datahub/search/event/tests/test_views.py]: I have added more teams.

File: datahub/search/event/tests/test_views.py (hunk 3 above)
Comment (elcct) [reply datahub/search/event/tests/test_views.py]: Indeed! Thanks :)

**Pull request merged**